dbt_platform_helper/commands/environment.py

```diff
@@ -39,97 +35,24 @@ def environment():
 @click.option("--vpc", type=str)
 def offline(app, env, svc, template, vpc):
     """Take load-balanced web services offline with a maintenance page."""
-    maintenance_page = MaintenancePageProvider()
-    maintenance_page.activate(app, env, svc, template, vpc)
+    try:
+        MaintenancePageProvider().activate(app, env, svc, template, vpc)
+    except PlatformException as err:
+        click.secho(str(err), fg="red")
+        raise click.Abort
 
 
 @environment.command()
 @click.option("--app", type=str, required=True)
 @click.option("--env", type=str, required=True)
 def online(app, env):
     """Remove a maintenance page from an environment."""
-    maintenance_page = MaintenancePageProvider()
-    maintenance_page.deactivate(app, env)
-
-
-def get_vpc_id(session, env_name, vpc_name=None):
```

Comment:
Probably worth flagging with a TODO where some of these methods might live? i.e. this looks to get the VPC for a given EC2 instance so it would make sense in an EC2 provider?